misc: code consistency and bot instance creation (#1482)
aiogram/aiogram: CHANGES/1482.misc.rst

> Enhanced code consistency and readability by refactoring imports, adjusting the base webhook URL, modifying bot instance initialization to utilize DefaultBotProperties, and updating router message handlers for improved consistency and readability.

Reviewer: ```suggestion
Improved code consistency and readability in code examples by refactoring imports, adjusting the base webhook URL, modifying bot instance initialization to utilize DefaultBotProperties, and updating router message handlers.
```
